stanford/playbooks/insights.yml

```diff
@@ -1,16 +1,36 @@
-- name: Deploy Analytics Insights
-  hosts: ~tag_Name_{{ machine }}_{{ env }}
-  become: True
+#!/usr/bin/env ansible-playbook
+---
+- include: ../playbooks/cluster.yml
+  vars:
+    CLUSTER_NAME: insights
+  when: redeploy | default(False) | bool
+
+- name: Deploy insights service
+  hosts: "tag_Cluster_{{ CLUSTER_NAME }}:&tag_todo_deploy:&tag_Deployment_{{ COMMON_DEPLOYMENT }}:&tag_Number_{{ CLUSTER_NUMBER }}"
   gather_facts: True
-  vars_files:
-    - "{{ COMMON_DEPLOYMENT_DIR }}/insights.yml"
+  become: True
   vars:
-    COMMON_ENABLE_DATADOG: False
-    COMMON_ENABLE_SPLUNKFORWARDER: False
+    CLUSTER_NAME: insights
+    migrate_db: 'yes'
+    disable_edx_services: false
+  vars_files:
+    - "{{ COMMON_DEPLOYMENT_DIR }}/common.yaml"
+    - "{{ COMMON_DEPLOYMENT_DIR }}/analytics-api.yml"
```

Comment:
With the refactoring we've made to the secrets, I think this role no longer needs to include `analytics-api.yml`, right?
There should be one reference to an `ANALTYICS_API_*` var, but it's being loaded from `common.yaml`.